handlers/api.go

```diff
@@ -1777,7 +1777,7 @@ func ApiValidatorBlsChange(w http.ResponseWriter, r *http.Request) {
 // @Param  latest_epoch query int false "The latest epoch to consider in the query"
 // @Param  offset query int false "Number of items to skip"
 // @Param  limit query int false "Maximum number of items to return, up to 100"
-// @Success 200 {object} types.ApiResponse{data=[]types.ApiValidatorBlsChangeResponse}
+// @Success 200 {object} types.ApiResponse{data=[]types.ApiValidatorBalanceHistoryResponse}
```

Comment:
*ApiValidatorBalanceHistoryResponse* is correct here but it is not used within the function itself, a new struct that is the same gets created instead.

Reply:
Good catch, fixed.